examples/resources/gateway_config_store.tf

```diff
@@ -0,0 +1,15 @@
+resource "konnect_gateway_config_store" "my_configstore" {
+  name  = "tf-config-store"
+  force = "true"
```

Comment:
@mheap is the force parameter still valid? I seem to be getting an error -
```
Error: Unsupported argument
│ 
│   on main.tf line 37, in resource "konnect_gateway_config_store" "my_configstore":
│   37:   force = true
│ 
│ An argument named "force" is not expected here.
```
which goes away once I remove it.

Reply:
I've updated the example. The `force` parameter is now set as a `const` and will always be sent